code/src/components/Artist.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export const Artist = (props) => {
+return(
+    <a href={props.artistDetails.external_urls.spotify} target="_blank" rel="noopener noreferrer" >
```

Comment:
Really like that you gave artists its own component, I think that is very semantically true when it comes to the structure of the website!